website/docs/r/private_dns_glb_monitor.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "ibm"
+page_title: "IBM : dns_glb_monitor"
+sidebar_current: "docs-ibm-resource-dns-glb-monitor"
+description: |-
+  Manages IBM Private DNS GLB monitor.
+---
+
+# ibm\_dns_glb_monitor
+
+Provides a private dns GLB monitor resource. This allows dns GLB monitor to be created,updated and deleted.
+
+## Example Usage
+
+```hcl
+
+resource "ibm_dns_glb_monitor" "test-pdns-glb-monitor-nw" {
```

Comment:
give full example as you have it in the test case

Reply:
Added